app/views/home/me.html.erb

```diff
@@ -0,0 +1,3 @@
+<div class="inset">
+ <h2>Welcome, <%= current_user.email_address %></h2>
+</div>
```

Comment:
interestingly, the header uses `session['user']['email']` instead of `current_user` so when `FORCE_USER_ID` is on, the header shows the authenticated user, and the /me page shows the effective user.